### PR TITLE
AppCenter - How to add access to partners

### PR DESCRIPTION
**Pull request by skywall**

### CODE REVIEW

**Comment by skywall** [comment]
Readable version: https://github.com/thefuntasty/developer-handbook/blob/feature/appcenter-add-partners-accounts/appcenter-how-to-add-partner-account.md

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 This handbook describes the best practices and guidelines we use for developing apps in The Funtasty. It's purpose is to serve both new members as an introduction and current members as a guide, particularly for the tedious tasks such as creating a new project. More broadly, general public can take a look on how we do things, learn, discuss and propose improvements.
 
 ## TOC
-- AppCenter - How to add partners' account
+- [AppCenter - How to add access to partners](https://github.com/thefuntasty/developer-handbook/blob/master//Users/skywall/Projects/developer-handbook/appcenter-how-to-add-partner-account.md)
```
Comment (mkj-is) [inline README.md]:
```suggestion
- [AppCenter - How to add access to partners](appcenter-how-to-add-partner-account.md)
```

File: appcenter-how-to-add-partner-account.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# AppCenter - How to add access to partners
+
+1. Create a new distribution group related to the app you want add access to (if already doesn't exists). 
+Distribution group management can be found at adress https://appcenter.ms/orgs/futured/apps/app-name/distribute/distribution-groups 
+(`app-name` should me replaced with a real name). 
+1. Add partners' emails as Testers of this distribution group.
```
Comment (mkj-is) [inline appcenter-how-to-add-partner-account.md]:
```suggestion
2. Add partner's emails as Testers of this distribution group.
```

File: appcenter-how-to-add-partner-account.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# AppCenter - How to add access to partners
+
+1. Create a new distribution group related to the app you want add access to (if already doesn't exists).
```
Comment (mkj-is) [inline appcenter-how-to-add-partner-account.md]:
```suggestion
1. Create a new distribution group related to the app you want add access to (if it already doesn't exist). 
```

File: appcenter-how-to-add-partner-account.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# AppCenter - How to add access to partners
+
+1. Create a new distribution group related to the app you want add access to (if already doesn't exists). 
+Distribution group management can be found at adress https://appcenter.ms/orgs/futured/apps/app-name/distribute/distribution-groups
```
Comment (mkj-is) [inline appcenter-how-to-add-partner-account.md]:
```suggestion
Distribution group management can be found at following address: https://appcenter.ms/orgs/futured/apps/app-name/distribute/distribution-groups 
```

File: appcenter-how-to-add-partner-account.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# AppCenter - How to add access to partners
+
+1. Create a new distribution group related to the app you want add access to (if already doesn't exists). 
+Distribution group management can be found at adress https://appcenter.ms/orgs/futured/apps/app-name/distribute/distribution-groups 
+(`app-name` should me replaced with a real name).
```
Comment (mkj-is) [inline appcenter-how-to-add-partner-account.md]:
```suggestion
(`app-name` should be replaced with a real app name). 
```

File: appcenter-how-to-add-partner-account.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+# AppCenter - How to add access to partners
+
+1. Create a new distribution group related to the app you want add access to (if already doesn't exists). 
+Distribution group management can be found at adress https://appcenter.ms/orgs/futured/apps/app-name/distribute/distribution-groups 
+(`app-name` should me replaced with a real name). 
+1. Add partners' emails as Testers of this distribution group.
+1. Add created distribution group to `$APP_CENTER_DISTRIBUTION_GROUPS` environment variable
```
Comment (mkj-is) [inline appcenter-how-to-add-partner-account.md]:
```suggestion
3. Add created distribution group to `$APP_CENTER_DISTRIBUTION_GROUPS` environment variable.
```

File: appcenter-how-to-add-partner-account.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+# AppCenter - How to add access to partners
+
+1. Create a new distribution group related to the app you want add access to (if already doesn't exists). 
+Distribution group management can be found at adress https://appcenter.ms/orgs/futured/apps/app-name/distribute/distribution-groups 
+(`app-name` should me replaced with a real name). 
+1. Add partners' emails as Testers of this distribution group.
+1. Add created distribution group to `$APP_CENTER_DISTRIBUTION_GROUPS` environment variable 
+to `enterprise` workflow at Bitrise. Distribution groups should be comma separated, e.g. 
+`"Collaborators,NewGroup"`. 
+1. Enjoy.
```
Comment (mkj-is) [inline appcenter-how-to-add-partner-account.md]:
```suggestion
4. Enjoy.
```

**Pull request merged**